### PR TITLE
Fix : Typos in README.md #5

### PR DESCRIPTION
**Pull request by Prity25-coder**
Fixed all the typos.
This PR will close #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +57,7 @@ However, the project contains **intentional bugs** that provide a good learning
 
 6. Start the development server:
     ```bash
-    npm start
+    npm run start
```
Comment (Ezio-Sarthak) [inline README.md:60]: `npm start` also works for me. maybe we can create an `or` here for understanding purpose

File: README.md [hunk 2]
```diff
@@ -70,14 +74,14 @@ However, the project contains **intentional bugs** that provide a good learning
     git push -u origin <your_branch_name>
     ```
 
-11. Create a `Pull-Request` to develop!
+11. Create a `Pull-Request` to the main branch(main branch of the original repo) from your forked repo branch
```
Comment (Ezio-Sarthak) [inline README.md:77]: a space b/w branch and `(` would look clean 👍

File: README.md [hunk 3]
```diff
@@ -88,9 +92,10 @@ ecommerce-product-page/
 │   │   ├── Cart.js          # Component for displaying the shopping cart
 │   │
 │   ├── App.js               # Main app component
-│   ├── index.js             # Entry piont for the app
+│   ├── index.js             # Entry point for the app
 │   ├── styles.css           # Styling for the components
 │
+|-- .gitignore               # To ignore the files and folders from git
```
Comment (Ezio-Sarthak) [inline README.md:98]: can we use an underscore here instead of hyphen to align with the rest of the chart?

**Review by Ezio-Sarthak — CHANGES_REQUESTED** [review]
Thanks for working on this @Prity25-coder . I've added some comments 👍

File: README.md (hunk 1 above)
Comment (Prity25-coder) [reply README.md:64]: Addressed

File: README.md (hunk 2 above)
Comment (Prity25-coder) [reply README.md]: Addressed

File: README.md (hunk 3 above)
Comment (Prity25-coder) [reply README.md]: Addressed

**Review by Ezio-Sarthak — APPROVED** [review]
Thanks for updating the PR @Prity25-coder !

This looks good to go! Approving this 👍

**Comment by Prity25-coder** [comment]
Conflict resolved.

**Comment by Prity25-coder** [comment]
Closing this PR, because some un-necessary commits are there.

- Raised another PR with same changes #42

**Comment by Ezio-Sarthak** [comment]
@Prity25-coder as mentioned earlier, kindly update the same PR by deleting those commits, please refrain from closing PRs. It will be difficult to track progress in those cases.

**Pull request closed without merging**